common4j/src/main/com/microsoft/identity/common/java/controllers/CommandDispatcher.java

```diff
@@ -85,13 +86,16 @@
 public class CommandDispatcher {
 
     private static final String TAG = CommandDispatcher.class.getSimpleName();
+    private static final int INTERACTIVE_REQUEST_THREAD_POOL_SIZE = 1;
     private static final int SILENT_REQUEST_THREAD_POOL_SIZE = 5;
-    private static final ExecutorService sInteractiveExecutor = Executors.newSingleThreadExecutor();
+    private static final ExecutorService sInteractiveExecutor = Executors.newFixedThreadPool(INTERACTIVE_REQUEST_THREAD_POOL_SIZE);
```

Comment:
I believe this returns a ThreadPoolExecutor... which has a getActiveCount method... I mention this because I'm not sure we should keep track of interactive account.... unless we're planning to support more than one in the future.... and I think at that point we want only 1 interactive request per Android Task....

Reply:
getActiveCount doesn't tell you what you think it might.  There *will* be race conditions where the active count is zero and there is a thread in queue.